_rest_routes.inc.php

```diff
@@ -355,13 +355,28 @@
         $data = (array) RestConfig::getPostData((file_get_contents("php://input")));
         return (new AuthRestController())->authenticate($data);
     },
+    "POST /fhir/Patient" => function () {
+        RestConfig::authorization_check("patients", "demo");
+        $data = (array)(json_decode(file_get_contents("php://input"), true));
+        return (new FhirPatientRestController())->post($data);
+    },
+    "PUT /fhir/Patient/:id" => function ($pid) {
+        RestConfig::authorization_check("patients", "demo");
+        $data = (array)(json_decode(file_get_contents("php://input"), true));
+        return (new FhirPatientRestController())->put($id, $data);
+    },
+    "PATCH /fhir/Patient/:pid" => function ($id) {
+        RestConfig::authorization_check("patients", "demo");
+        $data = (array)(json_decode(file_get_contents("php://input"), true));
+        return (new FhirPatientRestController())->put($id, $data);
```

Comment:
quick likely dumb question, why do the PUT/PATCH above both not use `:id` and `$id`?

Reply:
Hey @bradymiller they should be consistent! I'll update the routes.